### ISSUE TITLE
Category overview throws exception when setting default range to > 1 month.

### ISSUE THREAD

**Issue by boerbiet** [JC5/firefly-iii#170]
In my preferences I set the range to f.e. 3 months to see what that was like. Navigating to categories and clicking a category for an overview of it returns the following error:

```
FireflyException in Navigation.php line 262:
Cannot do subtractPeriod for $repeat_freq "3M"
```

It'll say 6M when selecting six months.

The stack trace (first 10 lines only):

```
in Navigation.php line 262
at Navigation->subtractPeriod(object(Carbon), '3M', '1') in compiled.php line 6031
at Facade::__callStatic('subtractPeriod', array(object(Carbon), '3M', '1')) in CategoryController.php line 226
at Navigation::subtractPeriod(object(Carbon), '3M', '1') in CategoryController.php line 226
at CategoryController->show(object(SingleCategoryRepository), object(Category))
at call_user_func_array(array(object(CategoryController), 'show'), array(object(SingleCategoryRepository), 'category' => object(Category))) in compiled.php line 8873
at Controller->callAction('show', array(object(SingleCategoryRepository), 'category' => object(Category))) in compiled.php line 8942
at ControllerDispatcher->call(object(CategoryController), object(Route), 'show') in compiled.php line 8922
at ControllerDispatcher->Illuminate\Routing\{closure}(object(Request))
at call_user_func(object(Closure), object(Request)) in compiled.php line 9593
```

**Comment by JC5** [comment]
I know where the error is, thanks for the stack trace! Secretly, I was
afraid this would happen. Hence the message, "cannot do...".

I will get a fix out tomorrow, which is in about ~12hrs.

**Comment by JC5** [comment]
Fixed in 3.7.1
